### ISSUE TITLE
IncorrectGroupOrder fix removes class attributes

### ISSUE THREAD

**Issue by krns** [slevomat/coding-standard#979]
Hi guys,

I have a laravel 6 application and php 7.4.4 running and just upgraded `slevomat/coding-standard` from 6.0.8 to 6.2.0. After that update phpcs reports a lot of `IncorrectGroupOrder` errors. This report is totally right, because I have a lot of protected or public attributes or methods in my models that are not in the correct order.

When I am trying to fix these issues with `phpcbf`, the methods are placed in the  correct order but the class attributes like the following are removed entirely..

```php
    /**
     * The attributes that should be hidden for arrays.
     *
     * @var array
     */
    protected $hidden = [
        'password',
        'remember_token',
    ];
```

**Comment by kukulich** [comment]
I'm not able to reproduce it. Everything works as expected: https://github.com/slevomat/coding-standard/commit/250ee8ebd8edcecd08a26ee82d1dfe7ed3d7d89f
